.github/workflows/ci.yml

```diff
@@ -48,8 +48,10 @@ jobs:
         run: npm ci
 
       # Act
-      - name: Check types
+      - name: Check codebase types
         run: npm run type-check
+      - name: Check Cypress types
+        run: npm run type-check:cypress
```

Comment:
I had to split the types validation into two commands, so now we need to run those 2 here.